src/soroban_rpc/mod.rs

```diff
@@ -0,0 +1,135 @@
+use jsonrpsee_core::{client::ClientT, rpc_params};
```

Comment:
I think you could name this mod simply `rpc`. No need to echo soroban, there's only one rpc server.

Reply:
maybe rpc_client as well tbh